openssl-builder/Dockerfile

```diff
@@ -13,19 +13,15 @@ RUN DEBIAN_FRONTEND=noninteractive apt-get update -y && \
     perl \
     wget \
     gnupg2
-RUN wget --quiet https://www.openssl.org/source/openssl-fips-$OPENSSL_FIPS_VER.tar.gz \
+COPY openssl*.key /
```

Comment:
Same here - explicitly import them rather than use globs for security

Reply:
Done